src/tray.js

```diff
@@ -61,14 +63,38 @@ exports.create = function(config) {
         },
     ]);
 
+    blinkEnabled = config.blinkEnabled;
+
     const defaultIcon = nativeImage.createFromPath(config.icon_path);
 
     trayIcon = new Tray(defaultIcon);
     trayIcon.setToolTip(config.brand);
     trayIcon.setContextMenu(contextMenu);
     trayIcon.on('click', toggleWin);
 
+
     let lastFavicon = null;
+
+    let blinkInterval = null;
+    const currentIcon = {icon: defaultIcon};
+    const emptyIcon = nativeImage.createFromBitmap(Buffer.alloc(4, 0), {width: 1, height: 1});
+    ipcMain.on('startBlinking', function() {
```

Comment:
These listeners are never removed.

Reply:
I put together the blink code into a separate object and added cleanup on tray icon destroy.